NorthstarDedicatedTest/squirrel.cpp

```diff
@@ -447,4 +460,29 @@ template <ScriptContext context> void ExecuteCodeCommand(const CCommand& args)
 		g_UISquirrelManager->ExecuteCode(args.ArgS());
 	else if (context == ScriptContext::SERVER)
 		g_ServerSquirrelManager->ExecuteCode(args.ArgS());
+}
+
+SQRESULT SQ_DevFuncStub(void* sqvm)
+{
+	spdlog::info("Blocked execution of squirrel developer function for security reasons. To re-enable them use start parameter "
```

Comment:
Maybe warn instead. Shouldn't usually be hit and that makes hitting it (for one reason or another) more obvious.

```suggestion
	spdlog::warn("Blocked execution of squirrel developer function for security reasons. To re-enable them use start parameter "
```